source/driverRFPhy.c

```diff
@@ -476,7 +473,7 @@ void rf_off(void)
         }
         rf_if_change_trx_state(TRX_OFF);
         rf_if_enable_slptr();
-        rf_flags_clear(~RFF_ON);
+        rf_flags_clear(RFF_ON);

```

Comment:
This used to clear "everything except ON", now it clears only ON.

I think logically this function should be explicitly clearing ON|TX|RX|CCA - it is shutting the chip down (rf_flags_reset?) Then callers don't need to do that.